iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -1537,6 +1544,15 @@ public boolean shouldOverwrite(org.apache.hadoop.hive.ql.metadata.Table mTable,
     return COPY_ON_WRITE.equalsIgnoreCase(mode);
   }
 
+  @Override
+  public void addResourcesForCreateTable(Map<String, String> tblProps, HiveConf hiveConf) {
+    String metadataLocation = tblProps.get(BaseMetastoreTableOperations.METADATA_LOCATION_PROP);
+    if (StringUtils.isNotEmpty(metadataLocation)) {
```

Comment:
it would be empty in general unless someone provides the the metadata location as part of the create command. like in normal create command it would be empty
create table ice (schema) stored by iceberg;
it would be empty here.
create table ice TBLPROPERTIES(metadata-location='') stored by iceberg;
we handle this case, & if it is provided we do the logic

Reply:
may be I can change it to just ``null`` check, if it is empty iceberg should, if it doesn't resolve to anything